notes/src/App.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from 'react';
+import './App.css';
+import { Route, withRouter } from 'react-router-dom';
+import axios from 'axios';
+
+import Navigation from './components/Navigation';
+import Notes from './components/Notes';
+import Note from './components/Note'
+import NewForm from './components/NewForm';
+import EditForm from './components/EditForm';
+
+class App extends Component {
+  constructor(props) {
+    super(props) 
+
+    this.state = {
+      notes: [],
+      title: '',
+      textBody: '',
+      updateTitle: '',
+      updateTextBody: '',
+      deleteToggle: false,
+    }
+  }
+
+  componentDidMount() {
+    axios
+      .get('https://fe-notes.herokuapp.com/note/get/all')
+      .then(response => {
+        this.setState(() => ({ notes: response.data }));
+      })
+      .catch(error => {
+        console.error('Server Error', error);
+      });
+  }
+
+  // componentDidUpdate() {
+  //   axios
+  //     .get('https://fe-notes.herokuapp.com/note/get/all')
+  //     .then(response => {
+  //       this.setState(() => ({ notes: response.data }));
+  //     })
+  //     .catch(error => {
+  //       console.error('Server Error', error);
+  //     });
+  // }
+
+  addNote = () => {
+    axios
+        .post('https://fe-notes.herokuapp.com/note/create', {
+          title: this.state.title,
+          textBody: this.state.textBody
+        })
+        .catch(err => {
+          console.log(err)
+        })
+
+    this.setState({ title: '', textBody: '' })
+  }
+
+  deleteNote = (id) => {
+    axios
+        .delete(`https://fe-notes.herokuapp.com/note/delete/${id}`)
+        .then(response => {
+          // console.log(response.data)
+        })
+        .catch(err => {
+          console.log(err)
+        })
+
+    this.setState();
+    this.props.history.push('/')
+  }
+
+  deleteToggleOn = () => {
+    this.setState({ deleteToggle: true })
+  }
+
+  deleteToggleOff = () => {
+    this.setState({ deleteToggle: false })
+  }
+
+  editNote = (id) => {
+    axios
+        .put(`https://fe-notes.herokuapp.com/note/edit/${id}`, {
+          title: this.state.updateTitle,
+          textBody: this.state.updateTextBody
+        })
+        .catch(err => {
+          console.log(err)
```

Comment:
in the .catch try using an error response instead of just a console.log When there is an error nothing will be displayed for the end user because console.log is only the terminal log for the dev

Reply:
Definitely. We never really addressed that till we started working on the backend so it's definitely something I do now.